samples/certs/README.md

```diff
@@ -7,5 +7,14 @@ Instructions are available [here](https://istio.io/docs/tasks/security/cert-mana
 The included sample files are:
 
 - `root-cert.pem`: root CA certificate.
-- `ca-cert.pem` and `ca-cert.key`: Citadel intermediate certificate and corresponding private key.
+- `ca-[cert|key].pem`: Citadel intermediate certificate and corresponding private key.
 - `cert-chain.pem`: certificate trust chain.
+- `workload-foo-[cert|key].pem`: workload certificate and key for URI SAN `spiffe://trust-domain-foo/ns/foo/sa/foo` signed by `ca-cert.key`.
```

Comment:
should this be in user facing samples or `tests/testdata/certs`? Already has a lot of certs, but maybe not sufficient for these tests

Reply:
I prefer to put these along with the signing certs (ca-cert) together to make it easier to manage them together. It's a lot certs but that shouldn't cause any confusions, the user doesn't even need to know these workload certs if they just follow the tasks on istio.io, and they could also use these workload certs for testing purposes.